integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -48,20 +47,30 @@ use near_primitives::views::{FinalExecutionStatus, QueryRequest};
 #[builder(pattern = "owned", build_fn(skip))]
 #[allow(unused)]
 struct TestReshardingParameters {
-    chunk_ranges_to_drop: HashMap<ShardIndex, std::ops::Range<i64>>,
+    base_shard_layout_version: u64,
     num_accounts: u64,
+    num_clients: u64,
+    num_producers: u64,
     num_validators: u64,
```

Comment:
Can you add comments for the validators and producers please? Depending on contexts validators may mean all the BPs, CPs and CVs but here it's only about CV-only.